apps/docs/content/components/Breadcrumbs/react.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Breadcrumbs
+source: https://github.com/primer/brand/tree/main/packages/react/src/Breadcrumbs/Breadcrumbs.tsx
+storybook: '/brand/storybook/?path=/story/components-breadcrumbs--playground'
+description: Use breadcrumbs to enable wayfinding through navigational links
```

Comment:
The overview page has a different description. Can we make them consistent?

Suggestion to go with your version...

```suggestion
description: Breadcrumbs display the current page within the site's hierarchy, allowing users to easily navigate to a different level.
```

Only reservation is using the term 'site' as the use-case for this isn't just limited to websites alone